drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fw_B/rpu_fw_patches.h

```diff
@@ -8,7 +8,7 @@
  * @brief Common structures and definations.
  */
 #ifdef CONFIG_NRF700X_RADIO_TEST
-const unsigned char wifi_nrf_umac_patch_pri_bimg[] = {
+const unsigned char __aligned(4) wifi_nrf_umac_patch_pri_bimg[] = {
```

Comment:
Are these changes needed any longer?

Reply:
Not really, but this is good to have in future in case we do things differently i.e., directly use this array, but I need to use GCC attribute here instead of Zephyr define, will update.